istio-control/istio-discovery/templates/enable-mesh-mtls.yaml

```diff
@@ -33,6 +33,7 @@ spec:
     tls:
       mode: ISTIO_MUTUAL
 ---
+{{ end }}
```

Comment:
I think this should cover both DRs, i.e move this `{{end}}` to the end of the next DR.

Reply:
sure. though this creates some delta between 1.4 and master then.